src/libLookup/Lookup.cpp

```diff
@@ -1880,11 +1880,6 @@ bool Lookup::ProcessSetStateDeltaFromSeed(const bytes& message,
     return false;
   }
 
-  if (!AccountStore::GetInstance().MoveUpdatesToDisk()) {
```

Comment:
Can you explain the removal here?

Reply:
Shouldn't store the states when received the delta while syncing, this was done unintentionally before.